src/components/Articles/CommentForm.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react"
+import axiosInstance from "../utilities/axios"
+
+const CommentForm = ({ articleId, onCommentSubmit }) => {
+  const [commentText, setCommentText] = useState("")
+  const [loading, setLoading] = useState(false)
+
+  const handleSubmit = async (event) => {
+    event.preventDefault()
+
+    if (!commentText.trim()) {
+      alert("Please enter a comment before submitting.")
+      return
+    }
+
+    try {
+      setLoading(true)
+
+      const response = await axiosInstance.post(
+        `/articles/${articleId}/comments`,
+        {
+          username: "happyamy2016",
+          body: commentText,
+        }
```

Comment:
Not a job for now but something you could come back to at a later date - perhaps you could hardcode a user in context or state instead and then access the stored value via template literals here?